.github/workflows/main.yml

```diff
@@ -31,8 +31,6 @@ jobs:
       uses: eskatos/gradle-command-action@v1
       with:
         arguments: check install
-    - name: Integration Test
```

Comment:
Had to delete this as this integration doesnt work with junit 5 as far as I know - we can revisit later if we want to